helper-persistence/src/main/java/org/example/persistence/auth/AuthPersistenceAdapter.java

```diff
@@ -0,0 +1,23 @@
+package org.example.persistence.auth;
+
+import lombok.RequiredArgsConstructor;
+import org.example.domain.auth.model.User;
+import org.example.domain.auth.spi.QueryAuthPort;
+import org.example.persistence.auth.mapper.UserMapper;
+import org.example.persistence.auth.repository.UserJpaRepository;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Component
+@RequiredArgsConstructor
+public class AuthPersistenceAdapter implements QueryAuthPort {
+    private final UserMapper userMapper;
+    private final UserJpaRepository userJpaRepository;
+
+    @Override
+    public Optional<User> getUserById(String id) {
+        return userMapper.toDomain(userJpaRepository.findByUserId(id).orElse(null));
+    }
```

Comment:
아래와 같이 행을 띄워서 코드 가독성을 높이는건 어떨까요
```suggestion
    public Optional<User> getUserById(String id) {
        return userMapper.toDomain(
            userJpaRepository.findByUserId(id).orElse(null)
        );
    }
```

아니 그보다 mapper 구현이 잘못 된 것 같은데
return 값이 optional이면 param 값도 optional로 들어가게해야 코드가 말이 되지 않을까요..
이걸 좀 늦게 알아차렸네요;;

Reply:
optional이 empty상태이면 null로 변경되어서 코드 자체는 정상작동하긴 합니다.
그런데 작동 과정이 한눈에 안들어오면 수정하긴 해야겠네요. 수정하는게 좋을까요?

Reply:
일단 가독성을 위해서 피드백 해주신대로 변경하겠습니다

Reply:
당연히 코드 중복을 줄이는 방향이 맞으니까 수정하는 편이 좋다 생각합니다

Reply:
알겠습니다! 그러면 수정해서 다시 올리도록 하겠습니다!

Reply:
근데 이게 Auth뿐만 아니라 다른 로직도 다 그렇게 처리되어있어서 수정하게 되면 좀 많은 변경사항이 발생할 것 같은데, 이 PR말고 따로 수정해서 해당 사항만 변경해 PR 올리겠습니다. 어떻게 생각하시나요?

Reply:
원래 여기 브랜치에서 수정하라고 말할려했는데 Major도 왜인지 persistence에 있네요? 다른 브랜치 만들어서 이것도 같이 PR 올리시믄 될듯 합니다